app/src/main/java/com/peacecorps/pcsa/MainActivity.java

```diff
@@ -262,7 +266,7 @@ public static void swapFragmentIn(FragmentActivity activity, Fragment fragment,
     }
 
     /**
-     * Populating the expandable list of the Navigation Drawer
+     * Populates the expandable list view of the Navigation Drawer
```

Comment:
Method comments follows a certain template which is discussed in detailed with the issue. Please read the discussion from the beginning . If you go through already accepted PRs for this issue, you can have a good idea what to be done.